src/main/java/Entities/UserMatches/MatchingAlgorithm.java

```diff
@@ -0,0 +1,7 @@
+package Entities.UserMatches;
+
+public interface MatchingAlgorithm{
+    void match();
+
+}
+
```

Comment:
Good use of clean architecture incase we want to introduce different matches in the future!